### ISSUE TITLE
Unknown Error - another reason found!

### ISSUE THREAD

**Issue by jamfx** [nextcloud/nextcloud-filelink#72]
Hi @ all,
I just found out:
If you use Lightning with an online Calendar in the same Nextcloud and of course you have provided an Username and Password for them. You HAVE TO use the SAME Account for the use of Filelink. Allthough you get no Error Messages while configuring FileLink and it connects to your NC, you will not be able to share the file an an E-Mail.

**Example:**
TB + Lightning with Accountname: MyUsernameForOnlineCalendarInNextCloud
FileLink -> Username and PW for: MyUsernameForOnlineCalendarInNextCloud 
Everything works fine.

NOT WORKING:
TB + Lightning with Accountname: MyUsernameForOnlineCalendarInNextCloud
FileLink -> Username and PW: MyUsernameForSharingFiles
**FILELINK NOT WORKING!**

I think this is an Thunderbird Issue. 
You can see the two Usernames and PWs stored in Thunderbirds Passwort-Management. 
But you can not use them for the Case I explained. 
So maybe this should be Mentioned as a Sidekick in the Guide as a Warning or a Hint. 
It also should be forced to Mozilla to fix this damn Issue. Same Problem with Using Online-Calendars. 
Short Example: 
If you want to use two or more Calendars from two or more users in one TB + Lightning, you are doomed. It's will not work, allthough the Credentials where successfully saved.

Thanks for your Attention.

**Comment by alfredbuehler** [comment]
> I think this iI think this is an Thunderbird Issue.

You're right, its a well known TB design weakness, that the password manager uses the server address to identify a credential entry.

Therefore opening an issue here, doesn't make real sense.
